deploy/kubernetes/sws.yaml

```diff
@@ -21,6 +21,27 @@ spec:
     app: sws
 ---
 apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: sws
+  labels:
+    app: sws
+  annotations:
+    ingress.kubernetes.io/rewrite-target: /
+spec:
+  backend:
+    serviceName: default-http-backend
+    servicePort: 80
+  rules:
+  - host: sws.info
+    http:
+      paths:
+      - path: /
```

Comment:
shouldn't we have some distinction routing to REST and to UI? or is it always has to go to the root? (root = /)

Reply:
The distinction is done inside the app - so this ingress just gets us into the app, we'll route /api and /console inside the app. The app is bound to a single host and port, so while I guess we could add separate ingresses here (sws.console and sws.api, versus just sws.info) not sure if that is what we want to do.

Reply:
I'm not sure that routing that is half configured inside the app and half outside of the app like here is a good direction for the architecture of the app.
@pilhuhn wdyt?

Reply:
OK, I am confusing things because I just took an example ingress and blindly added it.

I have since pushed changes to this ingress. No more confusion about default paths, rules for different paths/services.

There is one ingress now with one service, one port, going to a single service/port (our sws service). This should make more sense and is more clear with a smaller ingress definition.